src/frontend-samples/query-volume-app/VolumeQueryApp.tsx

```diff
@@ -0,0 +1,179 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from "react";
+import "@bentley/icons-generic-webfont/dist/bentley-icons-generic-webfont.css";
+import { EmphasizeElements, IModelApp, ScreenViewport, ViewClipClearTool, ViewClipDecorationProvider, ViewClipTool } from "@bentley/imodeljs-frontend";
+import { ClipMaskXYZRangePlanes, ClipPlaneContainment, ClipShape, ClipUtilities, ClipVector, Range3d } from "@bentley/geometry-core";
+import SampleApp from "common/SampleApp";
+import VolumeQueryUI from "./VolumeQueryUI";
+import { ColorDef, GeometryContainmentRequestProps } from "@bentley/imodeljs-common";
+import { BentleyStatus, Id64Array } from "@bentley/bentleyjs-core";
+import { PresentationLabelsProvider } from "@bentley/presentation-components";
+import { InstanceKey } from "@bentley/presentation-common";
+
+export interface SpatialElements {
+  insideTheBox: PhysicalElement[];
+  outsideTheBox: PhysicalElement[];
+  overlap: PhysicalElement[];
+}
+
+export interface SpatialElementsColors {
```

Comment:
Spatial could be problematic here too.  Maybe better to call it ClassificationColors.